eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from ..lib import manageevent as me
+from ..lib import readECF as rd
+import glob, os
+from ..lib import sort_nicely as sn
+import matplotlib.pyplot as plt
+from importlib import reload
+
+def fitJWST(eventlabel, workdir, speclc_dir, fit_par, run_par_file, meta):
+    # load savefile
+    if meta == None:
+        meta = me.load(speclc_dir + '/S4_' + eventlabel + '_Meta_Save.dat')
+
+    #read in run params
+    import eureka.S5_lightcurve_fitting.parameters as p
+    reload(p)
+    run_par=p.Parameters(param_file=run_par_file)
+    # Create directories for Stage 4 processing
+    files = glob.glob(os.path.join(speclc_dir + "/speclc", "*.txt"))
+    files = sn.sort_nicely(files)
+    if run_par.run_verbose.value:
+        print(files)
+    t0_offset = run_par.toffset.value#59694
+    for f in [files[0]]:
+        t_bjdtdb, flux, flux_err = np.loadtxt(f, skiprows=1).T
```

Comment:
Oh I see, so this is supposed to be a csv type output from Stage 4? Currently it outputs only a single .dat file

Reply:
All these parts were written before I started working on this - I can try and change them to connect to Stage 4 at some point.